lib/document.js

```diff
@@ -993,6 +993,12 @@ Document.prototype.overwrite = function overwrite(obj) {
  */
 
 Document.prototype.$set = function $set(path, val, type, options) {
+  if (utils.isPOJO(path) {
```

Comment:
```suggestion
  if (utils.isPOJO(path)) {
```

you are currently missing a ending `)`, so the tests fail

Reply:
Thanks, thats what I get for using the online editor.